packages/api/src/graphql/client.ts

```diff
@@ -11,62 +12,19 @@ import { group } from "./group.js";
 import { merkle } from "./merkle.js";
 import { ownership } from "./ownership.js";
 import { permission } from "./permission.js";
-import { proof } from "./proof.js";
-import axios, { AxiosResponse } from "axios";
-import { CookieJar } from "tough-cookie";
-import { wrapper } from "axios-cookiejar-support";
 
 const { ApolloClient, InMemoryCache, HttpLink, ApolloLink } = pkg;
 
-const isNode =
-  typeof process !== "undefined" &&
-  process.versions != null &&
-  process.versions.node != null;
-
-export interface IApiClient<T = any> {
-  api: BaseApiClient<T>;
-  db: ReturnType<typeof database>;
-  collection: ReturnType<typeof collection>;
-  index: ReturnType<typeof collectionIndex>;
-  doc: ReturnType<typeof document>;
-  user: ReturnType<typeof user>;
-  group: ReturnType<typeof group>;
-  ownership: ReturnType<typeof ownership>;
-  permission: ReturnType<typeof permission>;
-  merkle: ReturnType<typeof merkle>;
-  proof: ReturnType<typeof proof>;
-}
-
-export abstract class BaseApiClient<T = any> {
-  #client: InstanceType<typeof ApolloClient<any>> | undefined = undefined;
+export class ApiClient<T extends jose.JWTPayload> {
+  #client: InstanceType<typeof ApolloClient<any>>;
 
   context: Context<T> = new Context<T>();
 
-  public get apollo() {
+  public get client() {
```

Comment:
Shouldn't rename this property, it will create something strange like: `api.client.client`

Reply:
I agree, removed to apollo